tools/requirements.txt

```diff
@@ -0,0 +1 @@
+ml_dtypes    # for bf16 in numpy
```

Comment:
Previously, I had a PR in the MLIR community about this; it was approved and merged, but due to some post-CI issues, it was reverted. if they fixed their CI issues and re-merge the PR， we will not need to add the dependency here. 

In addition to this, we are also trying to use torch tensors to replace numpy  ndarray to avoid this dependency, which will be updated in Jialei's PR https://github.com/intel/graph-compiler/pull/161.

Reply:
> In addition to this, we are also trying to use torch tensors to replace numpy ndarray to avoid this dependency, which will be updated in Jialei's PR #161.

Let's do that first then

Reply:
hi, @kurapov-peter ,  the dependency has now been merged into the upstream, as seen here: https://github.com/llvm/llvm-project/pull/101271. Additionally, the recent update to our LLVM version includes that commit, and I have removed the dependency in this PR https://github.com/intel/graph-compiler/pull/106/commits/61fc0d6e2331bb233eb2ba342ee67c53e6a0dd58.